### DISCUSSION TITLE
Support for encoding animated AVIF?

### DISCUSSION THREAD

**Discussion by richardmonette** [libvips/libvips#3048] [General]
Sorry if this has been covered elsewhere, but does vips as of v8.13 support encoding animated AVIF?

**Comment by jcupitt** [comment]
Hello @richardmonette, yes, it's supposed to work, though I don't think anyone's used it much. I see:

```
$ vips copy 3198.gif[n=-1] x.avif
$ vips heifload x.avif x.gif --n=-1
```

It seems to forget the frame delays, and I'm not sure why the file's not been recognised properly, but it does make a working GIF.

**Reply by jcupitt** [reply]
... I found an issue with animated HEIF detection, they ought to be spotted correctly now, at least with git master libvips.

**Reply by jcupitt** [reply]
... I read a bit more about this. libvips will load and save many-frame AVIF images, but it can't handle the special AVIS animated HEIF type, since libheif does not support it.

**Comment by richardmonette** [comment]
Ah understood, thank you for taking the time to look into the current status 🙂
